Zend/zend_opcode.c

```diff
@@ -447,6 +447,9 @@ ZEND_API void destroy_zend_class(zval *zv)
 				if (prop_info->ce == ce) {
 					zend_string_release(prop_info->name);
 					zend_type_release(prop_info->type, /* persistent */ 1);
+					if (prop_info->attributes) {
+						zend_hash_release(prop_info->attributes);
```

Comment:
@iluuu1994 Looks like this makes 2 phar tests fail: coincidentally, https://github.com/php/php-src/pull/11980 has the same fix as yours and the same tests failures as well